.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,30 @@
+name: CI Build
+
+on:
+  - push
```

Comment:
Maybe this is better?

on:
  push:
    branches:
      - main
  pull_request:

Reply:
The lines formatting is gone...

Reply:
I just tried it out, but it did not re-run the tests after a push on this PR...
![image](https://user-images.githubusercontent.com/18247545/124074549-ef74b800-da43-11eb-9198-88cb62bc2abc.png)

Reply:
![image](https://user-images.githubusercontent.com/18247545/124075012-8b9ebf00-da44-11eb-8125-fc75a52f1b03.png)
(from https://docs.github.com/en/actions/reference/events-that-trigger-workflows)

Reply:
So we could configure it, but I think simply using push if fine